core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -884,6 +884,11 @@ private void internalStop() {
       try {
          stopCaches();
          globalComponentRegistry.getComponent(CacheManagerJmxRegistration.class).stop();
+      } catch (Throwable t) {
+         log.errorf(t, "Exception while shutdown of caches/JMX");
```

Comment:
I think we should have a separate try/catch for each action here so that we can better report the failures.